src/v2/view-builder/views/phone/ChallengeAuthenticatorDataPhoneView.js

```diff
@@ -39,9 +39,17 @@ const Body = BaseForm.extend(
       const extraCssClasses =
         this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login') ?
           'strong no-translate' : '';
+      let nicknameText = this.model.get('nickname') || '';
+      if (nicknameText.length > 20) {
+        nicknameText = nicknameText.substring(0, 20) + '...';
+      }
+      if (nicknameText !== '') {
+        nicknameText = ' (' + nicknameText + ')';
+      }
       // Courage doesn't support HTML, hence creating a subtitle here.
       this.add(`<div class="okta-form-subtitle" data-se="o-form-explain">${sendText}
-        <span ${ extraCssClasses ? 'class="' + extraCssClasses + '"' : ''}>${this.model.escape('phoneNumber')}</span>
+        <span ${ extraCssClasses ? 'class="' + extraCssClasses + '"' : ''}>
+        ${this.model.escape('phoneNumber')}${nicknameText}</span>
```

Comment:
`nicknameText` should only be added if `this.model.get('phoneNumber') !== loc('oie.phone.alternate.title', 'login')`, meaning the phoneNumber is like `XXX-XXX-5455` instead of the text `phone`.

Reply:
Addressed. Thanks